Airline_Web_API/Airline_Web_API/Controllers/UsersController.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Airline_Web_API.Models;
+using Airline_Web_API.DTOs;
+using Airline_Web_API.Services;
+using Airline_Web_API.Helpers;
+using Microsoft.AspNetCore.Authorization;
+
+using System.Net;
+
+namespace Airline_Web_API.Controllers
+{
+    [Route("api/users")]
+    [ApiController]
+    public class UsersController : ControllerBase
+    {
+        private readonly AccountService _accountService;
+        private readonly JwtService _jwtService;
+        public UsersController(AccountService accountService, JwtService jwtService)
+        {
+            _accountService = accountService;
+            _jwtService = jwtService;
+        }
+
+        [AllowAnonymous]
+        [HttpPost("authenticate")]
+        public async Task<ActionResult<Response<string>>> Authenticate([FromBody] AuthenticateModel model)
+        {
+            Response<User> authorizationResult = await _accountService.AuthenticateAsync(model);
+
+            if(authorizationResult.Data == null)
```

Comment:
What does this check actually mean?

Reply:
Class Response has field Data and if something while registration or authorization goes wrong, f.e. password is incorrect, Data will be null. If everything works correctly the value will be set to Data field. This way I check whether code worked out correctly and depending on the result the response will be sent